IAMOTZ/OOP-Warrior
# Wrote all the code to make the project work

#### What does this PR do
It adds all the codes necessary to allow the Warrior object function as expected
#### How should this be manually tested
- clone the repo 
- cd into the project directory
- Install all the dependency 
- RUN `npm test` 
- the output should be `14 passing`
